core/templates/dev/head/pages/base.html

```diff
@@ -31,7 +31,7 @@
 {%- endmacro %}
 
 <!DOCTYPE html>
-<html ng-app="oppia" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+<html ng-app="oppia" update-language lang="en" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
```

Comment:
Ah, ok, thanks! I'm OK leaving it like this for now, then, but we should probably check later (in the more advanced accessibility testing) whether this messes things up for screenreaders.

In that case, my only remaining comment is the technical one above.